some modifications to allow for easier reproducibility

antgonza opened the pull request.

wasade commented on ipynbs/Chasing_Platypus_via_MG-RAST.ipynb:
```diff
@@ -25,6 +25,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "<b>Before you continue: </b>This step is extremelly slow (~24 hrs.) and network intensive. Thus to facilitate this step you could download the files from April 22, 2015 [ftp://ftp.microbio.me/pub/platypus/mgrast_reports_042215.tgz](ftp://ftp.microbio.me/pub/platypus/mgrast_reports_042215.tgz). If you place them in the same folder that you have this IPython notebook the code will simply downl ...
```
notebook -> Notebook

btw, since you're modding this, possible to add a license mention in the top cell?

ElDeveloper commented on ipynbs/Chasing_Platypus_via_MG-RAST.ipynb:
```diff
@@ -270,8 +16,368 @@
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "<b>Before you continue: </b>This step is extremelly slow (~24 hrs.) and network intensive. Thus to facilitate this step you could download the files from April 22, 2015 [ftp://ftp.microbio.me/pub/platypus/mgrast_reports_042215.tgz](ftp://ftp.microbio.me/pub/platypus/mgrast_reports_042215.tgz). If you place them in the same folder that you have this IPython Notebook the code will simply dow ...
```
Since this is MarkDown, there's no need to do `<b>`, you can just simply do `**bold**`.

antgonza replied:
I like markdown ...

ElDeveloper commented on ipynbs/Chasing_Platypus_via_MG-RAST.ipynb:
```diff
@@ -270,8 +16,368 @@
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "This tutorial covers downloading all public MG-RAST datasets; parsing those files to search for a specific taxa of interest, in our case Platypus (Ornithorhynchus); and finding the geographycal location of the samples that match those samples, via Google. Note that you need [BeautifulSoup](http://www.crummy.com/software/BeautifulSoup/#Download/) installed."
```
This notebook not only queries for platypus, it also queries for Salmonella, Raphus and Thylacinus.

ElDeveloper commented on ipynbs/Chasing_Platypus_via_MG-RAST.ipynb:
```diff
@@ -270,8 +16,368 @@
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Once you are finish with the search we suggest double checking that the files generated have in the first column the actual name that was found and that it matches your search."
```
finish -> finished.

ElDeveloper replied:
Also this looks like a run-on sentence, maybe worth revising?

ElDeveloper commented on ipynbs/Chasing_Platypus_via_MG-RAST.ipynb:
```diff
@@ -270,8 +16,368 @@
+     "collapsed": false,
+     "input": [
+      "from operator import itemgetter\n",
+      "\n",
```
Extra blankline.

antgonza replied:
That's fine. I like to put 2 lines between imports and code (pep8)

ElDeveloper commented:
In general it would be awesome to note the size of the files hosted in microbio.me. Also, what do you think about having optional cells to download and unpack the files? That way those can wget, untar-zip and then put in the appropriate place for the notebook to work. I'm testing and I had to do this, so I assume others would have to do that as well.

ElDeveloper commented:
BTW, I'm still downloading the files and going through the notebook ...

ElDeveloper commented on ipynbs/Chasing_Platypus_via_MG-RAST.ipynb:
```diff
@@ -270,8 +16,368 @@
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "<b>Before you continue:</b> This step is not as slow as the previus one but still a bit slow (~1 hr). If you only want to reproduce the results from the paper you can download this file: [ftp://ftp.microbio.me/pub/platypus/mgrast_search_results.tgz](ftp://ftp.microbio.me/pub/platypus/mgrast_search_results.tgz)\n",
```
previous

ElDeveloper replied:
Sorry, previus -> previous.

ElDeveloper commented:
All looks good except from the comments I've made. Side-note, when I finished running the notebook I noticed there were fewer countries with platypus than the ones shown in the poster.

antgonza commented:
Addressed comments. I saw the same effect and I'm not sure who to
blame. Also note that in the previous version way more 'Undefined'
cause before we didn't have the 5 sec wait.

ElDeveloper commented on ipynbs/Chasing_Platypus_via_MG-RAST.ipynb:
```diff
@@ -25,6 +39,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "<b>Before you continue: </b>This step is extremelly slow (~24 hrs.) and network intensive. Thus to facilitate this step you could download the files from April 22, 2015 [ftp://ftp.microbio.me/pub/platypus/mgrast_reports_042215.tgz](ftp://ftp.microbio.me/pub/platypus/mgrast_reports_042215.tgz). If you place them in the same folder that you have this IPython Notebook the code will simply downl ...
```
Can you change the `<b>` to `**`.

ElDeveloper commented on ipynbs/Chasing_Platypus_via_MG-RAST.ipynb:
```diff
@@ -140,27 +196,36 @@
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<b>Note:</b> We suggest reviewing the resulting files, in specific check that the first column actually matches your query. Depending on the how broad the term is, it could have hits to other taxa."
```
Can you change the `<b>` to `**`?

ElDeveloper replied:
ping ... I'm not annoying :sob:

ElDeveloper commented on ipynbs/Chasing_Platypus_via_MG-RAST.ipynb:
```diff
@@ -100,7 +129,22 @@
    "metadata": {},
    "source": [
-    "Now that we have all MG-RAST files in our local folder we can search for any taxa of interest. Note that the HMTL filess that we downloaded in the previous step have all data information that MG-RAST provides from a sample, for example: multilevel taxonomic and functional assignments. Thus, in the next step you can search by any \"string\" that fits in those categories. The output of this se ...
+    "<b>Before you continue:</b> This step is not as slow as the previous one but still a bit slow (~1 hr). If you only want to reproduce the results from the paper you can download this file: [ftp://ftp.microbio.me/pub/platypus/mgrast_search_results.tgz](ftp://ftp.microbio.me/pub/platypus/mgrast_search_results.tgz)\n",
```
Can you change the `<b>` to `**`

ElDeveloper commented:
I just noticed that you committed `mgrast_reports_042215.tgz` into the repo (14 MB), would mind rebasing so you can remove that file from the commit or alternatively create a new PR? Whatever is easier for you.

ElDeveloper commented:
BTW, the notebook looks in great shape.
